code/datums/mind.dm

```diff
@@ -774,6 +774,17 @@
 	if(G)
 		G.reenter_corpse()
 
+//user: The mob that is suiciding
+/datum/mind/proc/suicide_act(mob/user)

```

Comment:
can you rename this to like `suicide_log` or something? for something that shares the name with the objects suicide act it's unintuitive given it's actual function.

Reply:
I thought about it and couldnt't really come up with anything to convey the meaning. I'll use yours.